docs/deployment.md

```diff
@@ -0,0 +1,58 @@
+This repository includes code for generating a self-zip archive
+of the code and all of it's Python dependencies. We use these
+archives to deploy the site to a Linux server.
+
+# What's an artifact?
+
+It's a zip file, that is executable and includes a Python "shebang" line. Here's a
```

Comment:
What's the importance of the fact that it includes a Python "shebang" line? This might be self-explanatory to most reading this doc, but it might also be helpful to elaborate on that.

Reply:
It's the thing that makes it self extracting-- if it didn't have a shebang line, it wouldn't be a valid executable.

Reply:
I re-arranged that section a bit, to describe that a little better.